firmware/controllers/trigger/decoders/trigger_structure.h

```diff
@@ -310,7 +310,7 @@ class TriggerFormDetails {
 	 * These angles are in event coordinates - with synchronization point located at angle zero.
 	 * These values are pre-calculated for performance reasons.
 	 */
-	angle_t eventAngles[PWM_PHASE_MAX_COUNT];
+	angle_t eventAngles[PWM_PHASE_MAX_COUNT * 2];
```

Comment:
I fixed this in my PR.

Reply:
then I'll just have to complain more about your PR so I beat you to master